spaceship/spec/spec_helper.rb

```diff
@@ -112,3 +112,25 @@ def mock_client_response(method_name, with: anything)
     end
   end
 end
+
+RSpec.shared_examples("common spaceship login") do |skip_tunes_login|
+  require 'fastlane-sirp'
+  let(:authentication_data) {
+    '8f30ce83b660f03abb0f8570c235e0e1e1d3860a222304acf18e989bdc065dc922a141e6da4563f0' \
+      '5586605b0e10535d875ca7e0fae7fe100cfe533374f29aaa803cdfb2c6194f458485e87f76988f6' \
+      'cddaa1829309438e1aa9ab652b17cfc081fff40356cb3af35c621e9f37ba6e2a03e6abac5a6bfe' \
+      '18ddb489412b7c56355292e6c355f8859270d04063b843d23c1ef7503c3c5dd2c56740101a3ef5' \
+      'bfec6bff1e6dc55e3f70840a83a95d7b3d20ab350d0472809ce87a4e3c29ed9685eb7721dc87ba' \
+      'bfadbd9e65e75d5df55547bcff98711ddeae7b8e1e6dbf529e96f7caa4b830b43575cddc52cebc' \
+      '39f9522f85cbf33ac35ee59f66f48109c12fbb78d'
+  }
+  let(:username) { 'spaceship@krausefx.com' }
+  let(:password) { 'so_secret' }
+
+  before {
+    allow_any_instance_of(SIRP::Client).to receive(:start_authentication).and_return(authentication_data)
+    allow_any_instance_of(SIRP::Client).to receive(:process_challenge).and_return("1234")
+
+    Spaceship::Tunes.login unless skip_tunes_login
+  }
+end
```

Comment:
Added a shared example that can be used in ALL of the tests that use spaceship to login that mocks the SIRP stuff. It gets used like `include_examples "common spaceship login"`